scripts/webpack/webpack.base.prod.js

```diff
@@ -3,26 +3,23 @@ const FaviconsWebpackPlugin = require('favicons-webpack-plugin')
 const ResolveTSPathsToWebpackAlias = require('ts-paths-to-webpack-alias')
 const CopyPlugin = require('copy-webpack-plugin')
 const { GenerateSW } = require('workbox-webpack-plugin')
-const HashedModuleIdsPlugin = require('webpack').HashedModuleIdsPlugin
-const { EnvironmentPlugin } = require('webpack')
-const { execSync } = require('child_process')
-// const SentryWebpackPlugin = require('@sentry/webpack-plugin')
-const { PATHS } = require('./constants')
-const { getVersionTag } = require('../release/utils')
-const hashFiles = require('../utils/hash-files')
 const path = require('path')
+const SentryWebpackPlugin = require('@sentry/webpack-plugin')
```

Comment:
So we're adding sentry source map to `webpack.base.prod.js`, does it mean that when we `yarn:build` in local machine, source map is still pushed to sentry?
Can we only do this when `CI = true` ?https://github.com/reapit/foundations/blob/master/.github/workflows/release-develop.yml#L16

Reply:
@vuhuucuong that's my concern, so let me find the way to remove build from PRs, and only accept in release develop and production.

Reply:
Good spot @vuhuucuong